python/src/equistore/block.py

```diff
@@ -205,6 +205,52 @@ def gradient(self, parameter: str) -> "Gradient":
         Get the gradient of the ``values`` in this block with respect to
         the given ``parameter``.
```

Comment:
Does this sound better to you?
```suggestion
        Get the gradient of the block ``values`` with respect to the
        given ``parameter``.      
```